benchmarks/run.sh

```diff
@@ -10,11 +10,12 @@ file=bench_outputs_$id.txt
 mkdir -p benchmarks/outputs
 echo -e "\e[36mOutputs will be saved to: benchmarks/outputs/$file\e[39m"
 echo -e "\e[35mStarting\e0\e[5m...\e[25m\e[39m"
-for i in {1..20}
+for i in {1..10}
```

Comment:
If the scope of the benchmarks was changed for performance reasons, please leave a comment here that the whole benchmark set ranges until 20